pkg/exporter/process.go

```diff
@@ -248,7 +243,7 @@ func (ep *ExportingProcess) createAndSendMsg(set entities.Set) (int, error) {
 
 	// Append the byte slices together to send on the exporter connection rather
 	// than copying the set buffer to message buffer again.
-	bytesSlice := append(msg.GetMsgBuffer().Bytes(), set.GetBuffer().Bytes()...)
+	bytesSlice := append(msg.GetMsgBuffer(), set.GetBuffer()...)
```

Comment:
could you try the following instead of append? Clearly the premise in the comment is wrong :)
https://github.com/golang/go/wiki/SliceTricks#copy

Reply:
Sure. Updated.